### PR TITLE
[WIP] Uniwap Markets

### PR DESCRIPTION
**Pull request by hayesgm**

### CODE REVIEW

File: poster/tests/uniswap_test.ts [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+      decimalsA: 18n,
+      decimalsB: 10n,
+      reserveA: e(10, 18),
+      reserveB: e(10, 10),
+      priceA: e(100, 6),
+      priceB: e(200, 6)
+    }
+  ].forEach(conf => {
+    test.only(conf.name, async () => {
+      let whit = await Whit.init({
```
Comment (coburncoburn) [inline poster/tests/uniswap_test.ts:83]: is this a philly cheesesteak reference?

Comment (hayesgm) [reply poster/tests/uniswap_test.ts:83]: It did come up in my head, yes, though also an oblique reference to Walt Whitman maybe?

Comment (coburncoburn) [reply poster/tests/uniswap_test.ts:83]: https://en.wikipedia.org/wiki/Walt_Whitman_House did you know walt whitman died just across the river in Camden?

**Comment by coburncoburn** [comment]
outdated and added elsewhere

**Pull request closed without merging**